packages/radio-box-group/src/RadioBox.styles.ts

```diff
@@ -0,0 +1,110 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { focusRing, fontFamilies, hoverRing } from '@leafygreen-ui/tokens';
+import { Size } from './types';
+
+export const radioBoxSizes: { [K in Size]: string } = {
+  [Size.Default]: css`
+    width: 169px;
+  `,
+
+  [Size.Compact]: css`
+    padding-right: 12px;
+    padding-left: 12px;
+  `,
+
+  [Size.Full]: css`
+    flex: 1;
+  `,
+};
+
+export const inputStyles = css`
+  opacity: 0;
+  position: absolute;
+  pointer-events: none;
+`;
+
+interface StateForStyles {
+  checked: boolean;
+  disabled: boolean;
+  size: Size;
+  showFocus: boolean;
+  darkMode: boolean;
+}
+
+export const getRadioDisplayStyles = ({
+  checked,
+  disabled,
+  size,
+  showFocus,
+  darkMode,
+}: StateForStyles) => {
```

Comment:
Not a huge fan of this pattern, but it works 👍

Reply:
I mostly copied these styles over to a new style file